rudder-server/SOURCES/mvn-like

```diff
@@ -28,3 +28,10 @@ else
 fi
 curl -s -o "${outfile}" "${url}"
 
+# Set a minimum size of 1Mb
+minimumsize=1000000
```

Comment:
1Mb is strictly 1024*1024 = 1048576, but I guess both values are close so it's fine ^^